api/v1/db/oracledb/member-dao.js

```diff
@@ -0,0 +1,59 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+const { serializeMembers, serializeMember } = require('../../serializers/members-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+
+/**
+ * @summary Return a list of members
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of members
+ */
+const getMembers = async (query) => {
+  const connection = await conn.getConnection();
+  try {
+    const rawMembers = await connection.execute('SELECT * FROM MEMBERS');
+    const rawMembersRow = rawMembers.rows;
+    const serializedMembers = serializeMembers(rawMembersRow, query);
+    return serializedMembers;
+  } finally {
+    connection.close();
+  }
+};
+
+/**
+ * @summary Return a specific member by unique ID
+ * @function
+ * @param {string} id Unique member ID
+ * @returns {Promise<Object>} Promise object represents a specific member
+ *                            or return undefined if term
+ *                            is not found
+ */
+const getMemberById = async (id) => {
+  const connection = await conn.getConnection();
+  try {
+    const sqlQuery = `
+    SELECT MEMBER_ID, MEMBER_NICKNAME, MEMBER_EMAIL, MEMBER_LEVEL, MEMBER_EXP_OVER_LEVEL 
+    FROM MEMBERS WHERE MEMBER_ID = :id
```

Comment:
I tried making this request `GET /members/a` and got a `500` response back. This was printed to the console:
```
SELECT MEMBER_ID, MEMBER_NICKNAME, MEMBER_EMAIL, MEMBER_LEVEL, MEMBER_EXP_OVER_LEVEL
    FROM MEMBERS WHERE MEMBER_ID = :id
     [ 'a' ]
Error: ORA-01722: invalid number
```
You define `memberId` to be a string in your openapi document so non-number strings are allowed. I think you want to change the type of this field in your database script to be a `VARCHAR2`

Reply:
I will change it to be a number.